UI/TiltAnalogSettingsScreen.cpp

```diff
@@ -41,6 +41,7 @@ void TiltAnalogSettingsScreen::CreateViews() {
 	settings->Add(new PopupSliderChoice(&g_Config.iTiltSensitivityX, 0, 100, co->T("Tilt Sensitivity along X axis"), screenManager(),"%"));
 	settings->Add(new PopupSliderChoice(&g_Config.iTiltSensitivityY, 0, 100, co->T("Tilt Sensitivity along Y axis"), screenManager(),"%"));
 	settings->Add(new PopupSliderChoiceFloat(&g_Config.fDeadzoneRadius, 0.0, 1.0, co->T("Deadzone Radius"), 0.01f, screenManager(),"/ 1.0"));
+	settings->Add(new PopupSliderChoiceFloat(&g_Config.fTiltDeadzoneSkip, 0.0, 1.0, co->T("Deadzone Skip"), 0.01f, screenManager(),"/ 1.0"));
```

Comment:
Hm, I think the name is the trickiest part of this setting, as usual... I'd almost call this a "deadzone boost" or "title base radius".  It's hard to explain properly alongside "sensitivity"...

Skip makes it too easy to think it'll do the opposite of what it does.

-[Unknown]

Reply:
How about "alter deadzone on tilt"?

Reply:
Changed with "Tilt Base Radius", using "deadzone" seem a source of confusion as if usually do the opposite indeed, need "lifezone" I guess :)

Reply:
I'm having trouble understanding the meaning of "tilt base radius": what is the base here?

Reply:
Base as a starting point, as when you move a little you start moving from the base radius (is a square, but still). TBH I'm not even sure if there is any non confusing way to say it.